src/statistics.jl

```diff
@@ -488,7 +515,33 @@ The algorithm is further described in [^Bačák2014].
     > doi: [10.1137/140953393](https://doi.org/10.1137/140953393),
     > arxiv: [1210.2145](https://arxiv.org/abs/1210.2145)
 """
-median(::Manifold, ::Any...)
+Statistics.median(
+    ::Manifold,
+    ::AbstractVector,
+    ::AbstractVector,
+    ::CyclicProximalPointEstimation,
+)
+
+"""
+    median(
+        M::Manifold,
+        x::AbstractVector,
+        [w::AbstractWeights,]
+        method::ExtrinsicEstimation;
+        extrinsic_method = CyclicProximalPointEstimation(),
+        kwargs...,
+    )
+
+Estimate the median of `x` using [`ExtrinsicEstimation`](@ref), i.e. by computing the median
+in the embedding and projecting the result back.
+You can specify a `extrinsic_method` to specify which mean estimation method to sue in the embedding,
```

Comment:
```suggestion
You can specify an `extrinsic_method` to specify which mean estimation method to use in the embedding,
```

Reply:
Thanks for noticing, this also occurs in the rephrased version of the mean (the first typo); the second is of course copy paste.